Detections/OfficeActivity/Malicious_Inbox_Rule.yaml

```diff
@@ -0,0 +1,34 @@
+id: 7b907bf7-77d4-41d0-a208-5643ff75bf9a
+name: Malicious Inbox Rule
+description: |
+  'Often times after the initial compromise the attackers create inbox rules to delete emails that contain certain keywords. 
+   This is done so as to limit ability to warn compromised users that they've been compromised. Below is a sample query that tries to detect this.
+  Reference: https://www.reddit.com/r/sysadmin/comments/7kyp0a/recent_phishing_attempts_my_experience_and_what/'
+requiredDataConnectors:
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity
+tactics:
+  - Defense Evasion
+relevantTechniques:
+  -  
+query: |
+
+ let timeframe = 14d;
```

Comment:
I don't think we need a detection for 14d, this should be 1d

Reply:
Done now.

Reply:
Thanks